app/views/users/index.html.erb

```diff
@@ -1 +1,13 @@
-<h1>List All Users</h1>
\ No newline at end of file
+<section class="container">
+  <%@users.each do |user| %>
+      <div class="user-info">
+          <div>
+              <img src=<%= user.photo %> >
+          </div>
+          <div>
+              <span>User Name:</span><%=user.name%><br>
+              <span>Number of Posts:</span><%=user.posts_counter%><br>
+          </div>
+      </div>
+  <%end%>
+</section>
```

Comment:
- [ ] Following the design pattern, kindly add a link that helps navigate to the `user#show` page